### DISCUSSION TITLE
Are there Keyboard Commands for +/- Layers?

### DISCUSSION THREAD

**Discussion by ghost** [sn4k3/UVtools#581] [Q&A] [answered]
As above really, are there any keyboard commands for navigating the layer view?

I was wondering because it would speed things up greatly if I could hit a key instead of running the mouse to the layer buttons in the bottom right of the window when say for example I'm done editing a layer but I need to quickly move onto the next one multiple times to remove a slicing artefact across many layers that is not an island like in the image.

Thanks.
![Screenshot 2022-10-11 at 17 55 13](https://user-images.githubusercontent.com/84204457/195662510-03e0d987-730c-464c-88d7-6e55b031261a.png)

**Comment by sn4k3** [comment, marked as answer]
You need to focus layer preview by click on layer image, then you can use Arrow up and arrow down to navigate up or down.
If you are out of focus, use CTRL + Up and CTRL + Down to have the same effect

**Reply by ghost** [reply]
Ok cool I did try some combinations with the arrow keys before I posted tbh but nothing happened so I assumed it I got it wrong or something. Gonna go try that again now. 
thanks for the reply!

**Reply by ghost** [reply]
> You need to focus layer preview by click on layer image, then you can use Arrow up and arrow down to navigate up or down.
> If you are out of focus, use CTRL + Up and CTRL + Down to have the same effect

Ok I figured it out - my computer just sucks & it’s slow & laggy as hell! It’s really old Haha 😂 🙈
